src/PhpBrew/Build.php

```diff
@@ -64,6 +64,8 @@ class Build implements Buildable
 
     public $osRelease;
 
+    public $osArch;
```

Comment:
```suggestion
    public string $osArch;
```

Reply:
The minimal PHP version about running the PHPBrew is `7.2` and it will be problematic when add the type hint for the public visible variable.